spec/geckoboard/payload_formatter_spec.rb

```diff
@@ -10,8 +10,8 @@ module Geckoboard
       formatter = PayloadFormatter.new(dataset)
 
       payload = formatter.format([
-        { 'birthday' => Date.new(2016, 1, 1) },
-        { 'birthday' => DateTime.new(2016, 1, 2, 10, 30) },
+        { 'birthday' => ::Date.new(2016, 1, 1) },
```

Comment:
Yeah, that's my general feels... how about we add `Field` to the type names?

e.g. `DateTimeField`